pdns/recursor_cache.cc

```diff
@@ -29,16 +29,16 @@ unsigned int MemRecursorCache::bytes() const
 {
   unsigned int ret=0;
 
-  for(cache_t::const_iterator i=d_cache.begin(); i!=d_cache.end(); ++i) {
+  for(const auto& i : d_cache) {
     ret+=sizeof(struct CacheEntry);
-    ret+=(unsigned int)i->d_qname.toString().length();
-    for(auto j=i->d_records.begin(); j!= i->d_records.end(); ++j)
-      ret+= sizeof(*j); // XXX WRONG we don't know the stored size! j->size();
+    ret+=(unsigned int)i.d_qname.toString().length();
+    for(const auto& record : i.d_records)
+      ret+= sizeof(record); // XXX WRONG we don't know the stored size! j->size();
```

Comment:
1. Is this comment still relevant?
2. `j->size()` has no meaning here

Reply:
given #6447, maybe just remove the comment here

Reply:
The comment is still relevant (agreed, `j` doesn't make sense anymore) until #6447 is merged so I'd rather keep it. I'll remove the  `j->size();` part.